### DISCUSSION TITLE
Reuse of the restart file

### DISCUSSION THREAD

**Discussion by sclee1717** [NASA-LIS/LISF#1542] [Q&A] [answered]
Hi all,

I'm using the Noah-MP 4.0.1 LSM, and I'm trying to run a new simulation with the restart file I generated before.
I want to ask, am I not possible to reuse the restart file for new runs with different input/output settings?
The previous domain was [0, 50] for lat and [67 141] for lon.

for example
Q1. can I ADD output variables listed in OUTPUT_LIST.TBL using the same restart file, generated without getting output for that specific variable? (can the restart file, generated from the simulation that doesn't give GPP as output variable, used to run a simulation that gives GPP as output variable; in another words, does the restart file have information on the variables not specified in the OUTPUT_LIST.TBL?)
Q2. can I use the different domain, for example [5, 20] for lat and [80, 110] for lon, using the restart file generated before ([0, 50] for lat and [67 141] for lon)?
Q3. same as Q2, but can I use different resolution?

As I tested, I found that all three cases "works" in the LIS (not giving any errors), but I'm not sure it is running correctly.
for Q1, I got the new LIS_HIST file with the new variable.
for Q2 and Q3, I got the below (guessing it is not properly working). The left one is the first run (larger domain) showing the domain for second run, and the right one is the second run.
![untitled](https://github.com/NASA-LIS/LISF/assets/60470915/f1b1e41c-4e0b-4588-b75b-e72898bf0de2)

Thanks in ahead!
Regards,
Lee

**Comment by dmocko** [comment, marked as answer]
Hi @sclee1717 

Let me start with Q2 and Q3.  No, you cannot restart a LIS simulation using different domain/resolution than the simulation that created the restart file.  You have to use the same exact domain/resolution for a restart file.  The run may not have crashed, but as you can see the output is unrealistic and should not be used.

For Q1: Yes, you can add (or remove) output variables in a restart simulation.  We do this often for long spin-ups to reach model equilibration.  We will only output model states such as soil moisture, SWE, TWS, etc.  Once the model has been spun-up, we will then use the restart file and run our simulation with many more output files listed in the OUTPUT.TBL file.

More generally, the LIS restart files contain all the model state information that is needed to continue the simulation.  The idea (for example) is that a two-day simulation should be totally identical to a one-day simulation for the first day, and then using a restart file for the second day.  The full list of restart variables in Noah-MP-4.0.1 LSM is given here:

https://github.com/NASA-LIS/LISF/blob/master/lis/surfacemodels/land/noahmp.4.0.1/NoahMP401_writerst.F90#L152-L204

**Reply by sclee1717** [reply]
Thank you @dmocko!
This is really clear.
